### PR TITLE
[New Rule][BBR] WRITEDAC Access on Active Directory Object

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Identifies the access on an object with WRITEDAC permissions. With the WRITEDAC permission, the user can perform a Write Discretionary Access Control List (WriteDACL) operation, which is used to modify the access control rules associated with a specific object within Active Directory. Attackers may abuse this privilege to grant themselves or other compromised accounts additional rights, ultimately compromising the target object, resulting in privilege escalation, lateral movement, and persistence.

### CODE REVIEW

**Review by Samirbous — APPROVED** [review]

File: rules_building_block/defense_evasion_write_dac_access.toml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+lateral movement, and persistence.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "WRITEDAC Access on Active Directory Object"
+note = """## Setup
+
```
Comment (brokensound77) [inline rules_building_block/defense_evasion_write_dac_access.toml:25]:
```suggestion
setup = """
```

**Review by brokensound77 — APPROVED** [review]
One comment then LGTM

File: rules_building_block/defense_evasion_write_dac_access.toml (hunk 1 above)
Comment (w0rk3r) [reply rules_building_block/defense_evasion_write_dac_access.toml:25]: Hmmm does this work?

Comment (w0rk3r) [reply rules_building_block/defense_evasion_write_dac_access.toml:25]: Hmmm it does, TIL

**Pull request merged**